native/packages/react-native-bpk-component-text/src/common-types.js

```diff
@@ -88,6 +89,7 @@ export const weightPropType = (
   componentName: string,
 ) => {
   if (!isWeightValid(props[propName], props.textStyle)) {
+    // eslint-disable-next-line no-console
```

Comment:
I don't know how CI was working without this before 🤷‍♂️

Reply:
It only gives an eslint warning, not an error, I think. Nice one for spotting it though!